android/src/com/frostwire/android/gui/fragments/preference/OtherFragment.java

```diff
@@ -110,7 +110,7 @@ private void setupClearIndex() {
                 @Override
                 public void onClick(View v) {
                     LocalSearchEngine.instance().clearCache();
-                    UIUtils.showShortMessage(getActivity(), R.string.deleted_crawl_cache);
```

Comment:
@gubatron the most important thing for me is to show the two toast messages in settings now. The "By default, one search engine needs to be checked at all times" and hopefully "Disconnected from BitTorrent. All BitTorrent transfers have stopped." 

I'm not sure if you can only use material style leaving everything else as is Gubatron (I would style all toasts now, btw). But it also seems like a lot additions. Is some java without adding layouts possible? ex. http://www.androidhive.info/2015/09/android-material-design-snackbar-example/ @grzesiekrzaca